### PR TITLE
Lars example file for sprint 1.3

### PR DESCRIPTION
**Pull request by larswik**
Example file for sprint 1.3

## File

Example file for sprint 1.3 first draft.

## Covered Concept Templates - Usages Included

1. Project Unit Assignment (1.1)
2. Project Representation Context (1.1)
3. Project Declaration (1.1)
4. Project Classification Information (1.2)
5. Spatial Decomposition (1.2)
6. Project Global Positioning - B (1.2)
7. Project Document Information (1.3)
8. Project Dataset Information (1.3)
9. Project Library Information (1.3)
10. Project Type Definitions (1.3)
11. Project Template Definitions

## Miscellaneous

Created with GeometryGymIFC_Core (0.2.11)

### CODE REVIEW

**Comment by pjanck** [comment]
I'd wait until the checker is online to approve. Looking through the example manually, I can't seem to spot any mistakes.

**Comment by larswik** [comment]
> I'd wait until the checker is online to approve. Looking through the example manually, I can't seem to spot any mistakes.

Of course. Since I am off duty tomorrow, I just thought that I could have a file ready for thursday.

File: files/project_team/IFCTunnelDep1.3.ifc
Comment (pjanck) [inline files/project_team/IFCTunnelDep1.3.ifc:28]: You might want to rename the project :)

File: files/project_team/IFCTunnelDep1.3.ifc
Comment (pjanck) [inline files/project_team/IFCTunnelDep1.3.ifc:51]: `Editors` field is empty, and thus an actor declaration by the project is missing as well.

File: files/project_team/IFCTunnelDep1.3.ifc
Comment (pjanck) [inline files/project_team/IFCTunnelDep1.3.ifc:61]:
```suggestion
#45= IFCPROPERTYSETTEMPLATE('3ruRe82Pv35A6pl1gt8lzj',#10,'Pset_GeoObsBoreholeConstruction',$,.PSET_TYPEDRIVENOVERRIDE.,'IfcBorehole',(#46));
```

**Review by pjanck — CHANGES_REQUESTED** [review]
I have a local checker instance up and running. I would like to hear your opinion on these errors reported by it before I go public.

File: files/project_team/IFCTunnelDep1.3.ifc
Comment (larswik) [reply files/project_team/IFCTunnelDep1.3.ifc]: Added an Editor. I don't quite understand why the IfcRelDeclares check is reporting an error? Currently the following is declared:
```
#33= IFCRELDECLARES('0uB3HDsab5m8q9MfuymxaG',#10,$,$,#11,(#31,#36,#45,#47));
#31= IFCBUILTSYSTEM('2qhchTsgr1_gWIcwPXP_ih',#10,'A system',$,$,.TUNNEL_SUPPORT.,$);
#35= IFCORGANIZATION('http://www.trafikverket.se','Trafikverket',$,$,$);
#36= IFCACTOR('2FCX85XmbF5hu3VP7$qYVD',#10,$,$,$,#35);
#45= IFCARCHELEMENTTYPE('03tHuUU5rC8AvZRAL_MSnn',#10,'Arch-123',$,$,$,$,$,'MyArchType',.USERDEFINED.);
#47= IFCPROPERTYSETTEMPLATE('1HnsV$PyzCJRmVHrw7NENU',#10,'Pset_GeoObsBoreholeConstruction',$,.PSET_TYPEDRIVENOVERRIDE.,'IfcBorehole',(#48));
```

**Review by pjanck — APPROVED** [review]
LFTM. The check somehow picks another file to check, thus failing ...

File: files/project_team/IFCTunnelDep1.3.ifc
Comment (pjanck) [reply files/project_team/IFCTunnelDep1.3.ifc]: Found the error - the checker was set up wrongly.

**Pull request merged**
